modules/st2-elastic-field/directive.js

```diff
@@ -0,0 +1,177 @@
+'use strict';
+angular.module('main')
+  .directive('st2ElasticField', [
```

Comment:
Well, now I see the benefit. By moving this logic to `st2FormTextField` we can make it more case-specific (like stop polluting the body).